cmake/FindAutotalks.cmake

```diff
@@ -0,0 +1,78 @@
+if(NOT AUTOTALKS_CRATON)
+    find_path(
+        AUTOTALKS_ROOT
+        NAMES "include/atlk/v2x_service.h"
+        PATH_SUFFIXES "autotalks_secton_api"
+        PATHS $ENV{HOME}
+        CMAKE_FIND_ROOT_PATH_BOTH # Seems like this is not needed in newer CMake
+    )
+
+    set(AUTOTALKS_INCLUDE_DIRS "${AUTOTALKS_ROOT}/include" "${AUTOTALKS_ROOT}/ref_src/include" "${AUTOTALKS_ROOT}/depend/device/include" "${AUTOTALKS_ROOT}/src/include" "${AUTOTALKS_ROOT}/src/core/include")
+    set(AUTOTALKS_LIBS_DIR ${AUTOTALKS_ROOT}/output.sec/x86/lib)
```

Comment:
Are there only x86 libraries? May we want to check `CMAKE_SYSTEM_PROCESSOR` to select the right directory?

Reply:
By `x86`, Autotalks code generation means that the system's native compiler is used (e.g. no cross-compilation) so in this case this will really be x86 on every platform (last year I tried SDK compilation on Raspberry PI and libraries were still in x86 folder). Autotalks documentation mentions that when Secton is controlled from Craton and we are cross-compiling for Craton, output folder is armv7-32. However, I did not try this possibility.